src/MomentoCacheClient.php

```diff
@@ -869,7 +871,7 @@ public function incrBy(string $key, int $value): Redis|int|false
         if ($result->asSuccess()) {
             return $result->asSuccess()->value();
         } else {
-            return false;
+            return MomentoToPhpRedisExceptionMapper::mapException($result->asError()->innerException());
```

Comment:
Can you add a method/overload to take as input the base error type from Momento? Then we can remove this repeated code to access the inner exception.